worker-message-prioritization-distribution-container/pom.xml

```diff
@@ -146,11 +146,38 @@
                     <containerNamePattern>%a-%t</containerNamePattern>
                     <verbose>true</verbose>
                     <images>
+                        <image>
```

Comment:
We don't want this image published, check you check we're not pushing it to artifactory?